src/components/Image/Image.driver.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+
+export interface ImageDriver extends BaseUniDriver {}
+
+export const imageDriverFactory = (base: UniDriver): ImageDriver => {
+  return {
+    ...baseUniDriverFactory(base),
```

Comment:
Do you think we should expose the methods of MediaImage's driver at the moment knowing this is a temporary solution until we've the final spec? 🤔

![image](https://user-images.githubusercontent.com/7141972/98693405-eb8f0b00-2378-11eb-8346-950399c6b384.png)

Reply:
Yeah, `click` is redundant. 
I think we can implement `getSrc`, `getAlt` and `getLoadStatus` for now.
Regarding `isLoaded` - seems like it's not implemented by the driver of ui-core's Image - not sure we actually need this if we already have `getLoadStatus`?

Reply:
I think you are right and the 3 methods that you wrote are enough. It seems like the getLoadStatus includes the (isLoaded, isError, and the isLoading). If I understood correctly :)

Reply:
but what about the `error` state?
We might want to support this, no?
ie:
```tsx
<img onError={this.props.onError} ...>
```
I'm assuming the `MediaImage` component also supports it

Reply:
I've planned to support error and image failure in different PR (out of the MVP scope).
Probably we'll want something like `onError` but in general we need to design the failure mode...